### PR TITLE
Deliver appointment confirmation email

### PR DESCRIPTION
**Pull request by AndrewVos**

### CODE REVIEW

File: spec/mailers/appointment_mailer_spec.rb [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        expect(body).to include('12:00am, 23 October 2016')
+        expect(body).to include("reference number, ##{appointment.id}")
+      end
+
+      it 'includes the guider' do
+        expect(body).to include(appointment.guider.name)
+      end
+    end
+  end
+  # it 'renders the subject' do
```
Comment (Guntrisoft) [inline spec/mailers/appointment_mailer_spec.rb]: ?

Comment (AndrewVos) [inline spec/mailers/appointment_mailer_spec.rb]: hehe, blatantly copying and pasting from `planner`

File: app/mailers/application_mailer.rb [hunk 2]
```diff
@@ -1,4 +1,4 @@
 class ApplicationMailer < ActionMailer::Base
-  default from: 'from@example.com'
+  default from: 'appointments@pensionwise.gov.uk'
```
Comment (benlovell) [inline app/mailers/application_mailer.rb]: Can we add the sender name here too? In the form `Pension Wise Appointments <appointments@pensionwise.gov.uk>`

File: spec/features/agent_manages_appointments_spec.rb [hunk 3]
```diff
@@ -10,6 +10,18 @@
       when_they_want_to_book_an_appointment
       and_they_fill_in_their_appointment_details
       then_that_appointment_is_created
+      and_the_customer_gets_an_email_confirmation
+    end
+  end
+
+  context 'Customer does not give an email' do
```
Comment (benlovell) [inline spec/features/agent_manages_appointments_spec.rb]: Can we flatten these contexts? When they only have a single context (and in a feature-style spec) there isn't any point separating them.

File: app/mailers/application_mailer.rb (hunk 2 above)
Comment (AndrewVos) [inline app/mailers/application_mailer.rb]: done

File: spec/features/agent_manages_appointments_spec.rb (hunk 3 above)
Comment (AndrewVos) [inline spec/features/agent_manages_appointments_spec.rb]: done

**Comment by benlovell** [comment]
You'll also need to add: `config.action_mailer.default_url_options = { host: ENV['APPLICATION_HOST'] }` to `production.rb`

**Comment by benlovell** [comment]
Also I forgot to mention previously but we have text templates in planner. We should over here too.

**Comment by AndrewVos** [comment]
@benlovell done and done

**Comment by benlovell** [comment]
Lovely 💯

**Pull request merged**